e2etests/test/registry.js

```diff
@@ -234,6 +234,63 @@ describe('Registry', function () {
     });
   });
 
+  it('Can create an edge and device scope relationship', function (done){
+    var registry = Registry.fromConnectionString(hubConnectionString);
+
+    var edgeDevice = {
+      deviceId: 'delete-me-' + uuid.v4(),
+      authentication: {
+        symmetricKey: {
+          primaryKey: new Buffer("1234567890qwerty").toString('base64'),
+          secondaryKey: new Buffer("ytrewq0987654321").toString('base64')
+        }
+      },
+      status: "enabled",
+      capabilities: {
+        iotEdge: true
+      }
+    };
+
+    var scopedDevice = {
+      deviceId: 'delete-me-' + uuid.v4(),
+      authentication: {
+        symmetricKey: {
+          primaryKey: new Buffer("1234567890qwerty").toString('base64'),
+          secondaryKey: new Buffer("ytrewq0987654321").toString('base64')
+        }
+      },
+      status: "enabled"
+    };
+
+    registry.create(edgeDevice, function(edgeCreateErr, edgeCreateResult) {
+      if (edgeCreateErr) {
+        done(edgeCreateErr);
+      } else {
+        debug('Created edge device: ', edgeCreateResult.deviceId);
+        assert(edgeCreateResult.capabilities.iotEdge, 'Created edge device does not contain correct capabilities.');
+        assert(edgeCreateResult.deviceScope, 'Created edge device does not contain a scope value.');
+        scopedDevice.deviceScope = edgeCreateResult.deviceScope;
+        registry.create(scopedDevice, function(scopedDeviceCreateError, scopedDeviceCreateResult) {
+          if (scopedDeviceCreateError) {
+            registry.delete(edgeCreateResult.deviceId, function() {
+              done(scopedDeviceCreateError);
+            });
+          } else {
+            debug('Created scoped device: ', scopedDeviceCreateResult.deviceId);
+            assert.equal(scopedDeviceCreateResult.deviceScope, scopedDevice.deviceScope, 'Created scoped device does not contain correct scope.');
+            scopedDeviceCreateResult.deviceScope = uuid.v4();
+            registry.update(scopedDeviceCreateResult, function(updateErr) {
+              assert(updateErr, 'Scoped device was incorrectly allowed to update its scope property.');
+              registry.delete(edgeCreateResult.deviceId, function () {});
```

Comment:
>function () {} [](start = 57, length = 14)

even if you don't want to fail we would benefit from logging here.
Failing to delete would probably expose an IoT Hub bug though... not sure why we'd ignore that.